core/server/worker/src/main/java/alluxio/worker/netty/BlockReadHandler.java

```diff
@@ -155,8 +155,9 @@ private void openBlock(BlockReadRequestContext context, Channel channel) throws
           try {
             BlockReader reader =
                 mWorker.readBlockRemote(request.getSessionId(), request.getId(), lockId);
+            String metricName = "BytesReadAlluxio";
```

Comment:
Should this be a constant somewhere? I don't see much point in making it a local var.

Reply:
talked offline